orga/translations/en.json

```diff
@@ -73,9 +73,10 @@
   },
   "charts": {
     "participants-by-stage": {
+      "loader": "Loading and sorting the success rates of each participant",
       "participants": "{count, plural, =0 {0 participants} =1 {1 participant} other {{count, number} participants}}",
       "percentage": "{percentage} %",
-      "title": "Distribution of participants by rating"
+      "title": "Success rates of each participant"
```

Comment:
on a les nouvelles traduction, car ça pouvait porter à confusion :
"Chargement de la répartition des participants par paliers" : Loading and sorting the participants grouped by success rates
"Répartition des participants par paliers" : Distribution of participants grouped by success rate

Reply:
Done